Add edge case test

mdonaka opened the pull request:
# Summary
Add edge case test.

# Purpose
Ensure operation by testing

# Contents
- Add edge case
  - send_share, execute_computation, etc...
- Add failure case
  - bts, send_share, etc...
# Testing Methods Performed
- small test
- medium tet
- CI

come2ry commented on packages/client/libclient-py/quickmpc/share.py:
```diff
@@ -98,7 +98,7 @@ def __sharize_1dimension_int(secrets: List[int], party_size: int = 3) \
         rnd: RandomInterface = ChaCha20()
         secrets_size: int = len(secrets)
-        max_val = max(secrets) * 2
+        max_val = (max(secrets)+1) * 2
```
@mdonaka 
What is the reason for +1, 0 avoidance?

mdonaka replied:
Yes. It is 0 avoidance.
The range is subject to reconsideration and will be considered separately.

arukuka reviewed (approved):
Thank you!  
👍

mdonaka commented on packages/client/libclient-py/tests/unit_tests/test_files/edge_data.csv:
```diff
@@ -0,0 +1,2 @@
+id,zero,int_max,int_min,float_min_plus,float_min_minus,string_max
+0,0,10000000000,-10000000000,0.00000000001,-0.00000000001,漢漢漢漢漢漢漢漢漢漢０漢漢００漢００漢アアアアアアアアアア漢漢漢漢・漢漢・漢漢漢漢漢漢アアアア＆アアアアアア漢漢漢漢漢漢漢漢漢漢漢（漢漢漢漢漢漢漢）漢
```
fix+string

mdonaka commented on packages/client/libclient-py/tests/unit_tests/test_files/edge_data.csv:
```diff
@@ -0,0 +1,2 @@
+id,zero,int_max,int_min,float_min_plus,float_min_minus,string_max
```
```suggestion
id,zero,int_max,int_min,float_min_plus,float_max_minus,string_max
```

come2ry commented on packages/client/libclient-py/tests/unit_tests/test_files/edge_data.csv:
```diff
@@ -0,0 +1,2 @@
+id,zero,int_max,int_min,float_min_plus,float_min_minus,string_max
+0,0,10000000000,-10000000000,0.00000000001,-0.00000000001,漢漢漢漢漢漢漢漢漢漢０漢漢００漢００漢アアアアアアアアアア漢漢漢漢・漢漢・漢漢漢漢漢漢アアアア＆アアアアアア漢漢漢漢漢漢漢漢漢漢漢（漢漢漢漢漢漢漢）漢
```
@mdonaka 

Is the range of int_max, int_min correct?
Is the range of float_min_plus, float_min_minus correct?
Is float_min_minus not float_max_minus?
If string_max represents the address of the longest string, then I think we can write the address as it is.

int_max, int_minのレンジは正しいか？
float_min_plus, float_min_minusのレンジは正しいか？
float_min_minusではなくfloat_max_minusではないか？
string_maxは最長文字列の住所を表しているのであれば住所をそのまま書いてしまっていいと思う

mdonaka replied:
@come2ry 
> Is the range of int_max, int_min correct?
Is the range of float_min_plus, float_min_minus correct?
Is float_min_minus not float_max_minus?

Thank you! I fix it!
From left to right: $10^{1200}, -10^{1200}, 10^{-50}, -10^{-50}$
> If string_max represents the address of the longest string, then I think we can write the address as it is.

I checked again and found that I had referred to an address that was not the longest in the first place.
The longest one could not be found, so it was changed to 161 Kanji characters as an alternative.
commit: c65e40edf331082b2f71406197674d9b65649bf9

mdonaka replied:
@come2ry 
Soryy.... In the test for this edge case, `assert` was missing and passed everything.
The current libclient-py does not support huge numbers, and the test no longer passes. Therefore, I have reverted to a small value.
7c6dbebba549c2393fd6618a2c09be3d63f850ab

come2ry replied:
I had missed it too...
Thanks!

come2ry commented:
絵文字は対応しますか？😄

mdonaka commented on packages/server/beaver_triple_service/utils/interceptor.go:
```diff
@@ -75,14 +74,6 @@ func AuthJWT(tokenString string) (*jwt_types.Claim, error) {
 		return nil, status.Error(codes.Internal, "failed claims type assertions")
 	}
 
-	// expのバリデーション
-	if claims, ok := token.Claims.(jwt.MapClaims); ok && token.Valid {
-		exp := time.Unix(int64(claims["exp"].(float64)), 0)
-		if time.Now().After(exp) {
-			return nil, status.Error(codes.Internal, "token is expired")
-		}
-	}
```
I deleted this because it was validated with `jwt.ParseWithClaims` and did not reach here 
https://github.com/golang-jwt/jwt/blob/b357385d3ee5c53c16db81addf9c6c8059c4cf15/parser.go#L105
https://github.com/golang-jwt/jwt/blob/b357385d3ee5c53c16db81addf9c6c8059c4cf15/validator.go#L90

come2ry replied:
Oh...
Thanks for noticing and correcting my mistake.

mdonaka commented:
> Do you support emoji? 

I don't think it is necessary to support it, but I added it anyway because it worked when I added it.
59a7840015d0b33076fdeab8e346913d7a08f659

come2ry commented:
LGTM

KotaTakahashi9320 reviewed (approved):
LGTM
